task16/index.js

```diff
@@ -197,14 +19,11 @@ const removeLoading = () => document.getElementById("loadingPlace").remove();
 
 const getFetchData = async () => {
   try {
-// const respons = await fetch(URL);
-    // if (!respons.ok) {
-    //   throw new Error("ただいまサーバー側で通信が壊れています");
-    // }
-    // const json = await respons.json();
-    // return json;
-    //*myjsonが繋がらないので一時的に下記の記述をしています。
-    const json =  URL;
+    const respons = await fetch(URL);
+    if (!respons.ok) {
+      throw new Error("ただいまサーバー側で通信が壊れています");
+    }
```

Comment:
The other day on Discord, Moriken-sensei told us that we should not throw ERROR here. Please check the thread on Issue 14 for more information.
```suggestion
    if (!respons.ok) {
      console.error("ただいまサーバー側で通信が壊れています");
    }
```

Reply:
I did not see the thread on Issue 14. Thanks for letting me know. I learned a lot.
2528c03